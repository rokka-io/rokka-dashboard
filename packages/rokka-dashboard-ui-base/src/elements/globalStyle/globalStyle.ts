import { createGlobalStyle } from 'styled-components';
import { SpacesGlobalStyle } from '../../identity/spaces/spaces';
import { FontGlobalStyle } from '../../identity/typography';

export const GlobalStyle = createGlobalStyle`
  ${SpacesGlobalStyle}
  ${FontGlobalStyle}
`;
