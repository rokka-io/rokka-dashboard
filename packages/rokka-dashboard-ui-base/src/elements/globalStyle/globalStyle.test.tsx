import { mount } from 'enzyme';
import React from 'react';
import { GlobalStyle } from './globalStyle';

describe('Elements', () => {
  describe('GlobalStyle', () => {
    it('renders the global style correctly.', () => {
      const component = mount(<GlobalStyle />);

      expect(component).toMatchSnapshot();
    });
  });
});
