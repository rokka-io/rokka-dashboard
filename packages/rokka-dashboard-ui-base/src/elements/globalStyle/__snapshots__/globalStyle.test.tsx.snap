// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Elements GlobalStyle renders the global style correctly. 1`] = `<GlobalStyleComponent />`;
